IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcGeoScienceModelType/Documentation.md

```diff
@@ -0,0 +1,2 @@
+The _IfcGeoScienceModelType_ provides the type information for _IfcGeoScienceModel_ occurrences.
+An _IfcGeoScienceModel_ represents a model of the geological structure as considered relevant for the project, as a base for the definition of building- and design-related geotechnical models, hydrogeological models and geo hazard models.
```

Comment:
Do you want the description of IfcGeoScienceModel on IfcGeoScienceModelType?

Reply:
Maybe not. I just looked at IfcPavementType as a template.